source/time.tex

```diff
@@ -9267,13 +9267,14 @@
 The \tcode{save} data member is extra information not normally needed
 for conversion between \tcode{local_time} and \tcode{sys_time}.
 If \tcode{save != 0min}, this \tcode{sys_info} is said to be on ``daylight saving'' time,
-and \tcode{offset - save} suggests what offset this \tcode{time_zone} might use
+and \tcode{offset - save} provides a non-authoritative suggestion of
+the offset this \tcode{time_zone} would use
 if it were off daylight saving time.
-However, this information should not be taken as authoritative.
-The only sure way to get such information
-is to query the \tcode{time_zone} with a \tcode{time_point}
+A correct offset for the \tcode{time_zone} off daylight saving time
+can be obtained by querying the \tcode{time_zone} with a \tcode{time_point}
 that returns a \tcode{sys_info} where \tcode{save == 0min}.
-There is no guarantee what \tcode{time_point} might return such a \tcode{sys_info}
+There is no guarantee
+what \tcode{time_point} (if any) returns such a \tcode{sys_info}
```

Comment:
I find this last sentence hard to understand (before and after this change).

> For any given `time_zone`, it is possible that there is no `time_point` that returns a `sys_info` not on daylight saving time. If such a `time_point` does exist, it is guaranteed not to be in the range \[`begin`,end`) of any `sys_info` on daylight saving time.

I also think "querying the `time_zone` could be more precise, by referring to the `get_info` member.

Reply:
I think any such larger rework is for C++23. Added to #4340.

Reply:
@jwakely If you're happy with the suggested approach (this change for C++20, something else for C++23), please can you approve or merge this PR?